homeassistant/components/waqi/sensor.py

```diff
@@ -0,0 +1,98 @@
+"""
+Sensor platform for the WAQI Component.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor/waqi/
+"""
+import logging
+from homeassistant.const import (ATTR_ATTRIBUTION, ATTR_TIME, ATTR_TEMPERATURE,
```

Comment:
Add a blank line between standard library, 3rd party and homeassistant import groups.